Dockerfile.ci

```diff
@@ -1242,7 +1242,7 @@ RUN bash /scripts/docker/install_mysql.sh prod \
     && chmod 0440 /etc/sudoers.d/airflow
 
 # Install Helm
-ARG HELM_VERSION="v3.9.4"
+ARG HELM_VERSION="v3.15.2"
```

Comment:
Actually - let's hold on with merging this one - seems that tomorrow they plan to release 3.15.3 bugfix release.

Reply:
Indeed. They kept their promise.